common/messages/sceneGame.ts

```diff
@@ -0,0 +1,12 @@
+import {TetrominoType} from "../TetrominoType"
```

Comment:
Should this file be renamed to sceneGameArena to match up with the file naming scheme you have?